<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cMainWindow</class>
 <widget class="QMainWindow" name="cMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReadySCOPE - Gestion des configurations</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/logo.png</normaloff>:/img/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageHome"/>
      <widget class="QWidget" name="pageSettings">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>399</width>
             <height>404</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>9</number>
            </property>
            <property name="rightMargin">
             <number>9</number>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Scaling</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <layout class="QFormLayout" name="formLayout_4">
                 <item row="0" column="0">
                  <widget class="QLabel" name="scalingALabel">
                   <property name="text">
                    <string>ScalingA :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QDoubleSpinBox" name="edtScalingA"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="scalingBLabel">
                   <property name="text">
                    <string>ScalingB</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QDoubleSpinBox" name="edtScalingB"/>
                 </item>
                 <item row="2" column="1">
                  <widget class="QCheckBox" name="chbScalingEnabled">
                   <property name="text">
                    <string>ScalingEnabled</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Format</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="displayFormatLabel">
                   <property name="text">
                    <string>DisplayFormat :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="edtDisplayFormat"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="binaryFormatLabel">
                   <property name="text">
                    <string>BinaryFormat :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="edtBinaryFormat"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>General</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="0">
                <layout class="QFormLayout" name="formLayout_2">
                 <item row="0" column="0">
                  <widget class="QLabel" name="inputDeviceLabel">
                   <property name="text">
                    <string>InputDevice :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="edtInputDevice"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="bufferSizeLabel">
                   <property name="text">
                    <string>BufferSize :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QSpinBox" name="edtBufferSize">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>500000</number>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="nanValueLabel">
                   <property name="text">
                    <string>NanValue :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QSpinBox" name="edtNanValue">
                   <property name="minimum">
                    <number>-32768</number>
                   </property>
                   <property name="maximum">
                    <number>32767</number>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="afterGlowLabel">
                   <property name="text">
                    <string>AfterGlow :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QLabel" name="refreshRateLabel">
                   <property name="text">
                    <string>RefreshRate :</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="1">
                  <widget class="QSpinBox" name="edtRefreshRate">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>50</number>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QRadioButton" name="radAfterGlowYes">
                     <property name="text">
                      <string>Yes</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="radAfterGlowNo">
                     <property name="text">
                      <string>No</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="layoutName">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSettingsOK">
            <property name="text">
             <string>OK</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>419</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGeneral">
    <property name="title">
     <string>Général</string>
    </property>
    <addaction name="actionSaveSettings"/>
    <addaction name="actionLoadSettings"/>
    <addaction name="actionSettingsPage"/>
    <addaction name="actionCloseApplication"/>
   </widget>
   <addaction name="menuGeneral"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCloseApplication">
   <property name="text">
    <string>Fermer l'application</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="text">
    <string>Charger les configurations</string>
   </property>
  </action>
  <action name="actionSaveConfigurtation">
   <property name="text">
    <string>Enregistrer une configuration</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>Enregistrer les données</string>
   </property>
  </action>
  <action name="actionClearData">
   <property name="text">
    <string>Nettoyer les données</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Arrêter</string>
   </property>
  </action>
  <action name="actionInitPort">
   <property name="text">
    <string>Initialiser</string>
   </property>
  </action>
  <action name="actionDisplayConnection">
   <property name="text">
    <string>Afficher la liste des connexions port série</string>
   </property>
  </action>
  <action name="actionHomePage">
   <property name="text">
    <string>Afficher la page d'accueil</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Sauvegarder les configurations</string>
   </property>
  </action>
  <action name="actionSettingsPage">
   <property name="text">
    <string>Afficher les configurations</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionCloseApplication</sender>
   <signal>triggered()</signal>
   <receiver>cMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
